packages/keystone-engine.vm/keystone-engine.vm.nuspec

```
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>keystone-engine.vm</id>
```

[nitpick] I think the name should be keystone. The project/tool is called keystone, but the Python library is called `keystone-engine` because there is another Python library with the name `keystone`. If you change it here, remember to change the file names too.
```suggestion
    <id>keystone.vm</id>
```